submodules/groups/groups.js

```diff
@@ -1046,6 +1046,8 @@ define(function(require) {
 					//formData = self.groupsCleanNameData(formData);
 
 					data = $.extend(true, {}, data, formData);
+					var baseGroupName = data.group.name + ' Base Group';
+					var ringGroupName = data.group.name + ' Ring Group';
```

Comment:
Since you are adding new occurrences of those two strings, an improvement over the old code would to be to create i18n key/values for this and replace it everywhere in this submodule so it looks like this instead:
```suggestion
					var ringGroupName = data.group.name + 'self.i18n.active().groups.ringGroup;
```

An even better solution would be to set a variable directly in the `i18n` file so that people can move it around (especially useful for languages that structure sentences the opposite way than English does):

```suggestion
					var ringGroupName = self.getTemplate({
						name: '!' + self.i18n.active().groups.ringGroup,
						data: {
							name: data.group.name
						}
					});
```
The corresponding i18n entry would look like this: `"ringGroup": "{{name}} Ring Group"`.

This is what we call a string template in Monster UI jargon (way different than ES6's template strings...). You can differentiate it from a regular template request because the result is not wrapped in a jQuery object.

Reply:
great!

Done!